person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def rentals(book, date)
+    Rental.new(book, date, self)
+  end
```

Comment:
- [ ] Kindly arrange your parameters to use the order you have in the Rental constructor.

  ```
  Rental.new(date, book, self)
  ```

Reply:
Very progressive advise!!